gradle/versions.gradle

```diff
@@ -28,7 +28,6 @@ ext.libVersions =
     protoc: '3.5.1',
     findbugs: '1.3.9',
     hikariCP: '3.4.1',
-    checkstyle: '6.18',
```

Comment:
AtlasDB has had JDK8 as minimum dependency for a while, and this old version of checkstyle isn't compatible with gradle-baseline's checkstyle rules, and this old version would breaks Idea Checkstyle integration, but now this all works properly.